src/components/Dashboard/Dashboard.jsx

```diff
@@ -20,8 +20,8 @@ export class Dashboard extends React.Component {
     }
 
     render() {
-        const showPlayerInteractions = (this.props.voteCard || this.props.word || (!this.props.playWord && this.props.playCard)) && !(this.props.winner || this.props.drawers.length > 1);
-        const showPlayWord = this.props.playWord && !this.props.word && !(this.props.winner || this.props.drawers.length > 1);
+        const showPlayerInteractions = (this.props.voteCard || this.props.word || !this.props.playedCardId) && (this.props.playCard || this.props.playedCardId) && !(this.props.winner || this.props.drawers.length > 1);
```

Comment:
This logic seems weird with the `!playedCardId` and `playedCardId` and stuff, is there any way you can neaten it up?